packages/oui-datagrid/src/datagrid.controller.js

```diff
@@ -313,6 +318,37 @@ export default class DatagridController {
         };
     }
 
+    getSelectedRows () {
+        return this.selectedRows.reduce((result, selected, index) => {
+            if (selected) {
+                result.push(this.displayedRows[index]);
+            }
+            return result;
+        }, []);
+    }
+
+    onSelectRow (index, selected) {
```

Comment:
`onRowSelection` ? Still feels wrong as this method is called even when the row is unselected

Reply:
onToggleSelectRow would be better i think

Reply:
In this case, `toggleRowSelection` seems more appropriate